src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -94,10 +104,26 @@ private static void OnProcessExit(object sender, EventArgs e)
             }
         }
 
+        private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
+        {
+            using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
+            LogLevel msalLogLevel = LogLevelMap[args.LogLevel];
+            if (_logger.IsLoggingEnabled(msalLogLevel))
+            {
+                string msgWithPii = string.Empty;
+                string msgNoPii = string.Empty;
+
+                // as of now, there will be no Pii.
```

Comment:
Please add a link to the GH issue

Reply:
Done.

Reply:
@SameerK-MSFT Was just thinking about this again. I may have misheard today in standup. So is this correct - errors from runtime time **do not** have PII; they are scrubbed? (I think I just heard you say the opposite).